pkg/config/config.go

```diff
@@ -140,8 +141,15 @@ func checkFilePerms(files ...string) error {
 			continue
 		}
 
-		if fInfo.Mode() != 0600 {
-			return errors.E(op, f+" should have 0600 as permission")
+		if runtime.GOOS == "windows" {
```

Comment:
do we really need to check file permissions on a config file in the first place? Why is this important?

Reply:
@marwan-at-work To avoid credentials to be leaked out to the world. It is easy for users to expose files to the world. The proxy can check if files specified are only group readable and owner writeable but not world readable & writeable, we can avoid that.

Reply:
LGTM